shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -419,6 +431,17 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   return [self runWithEntrypoint:entrypoint libraryURI:nil];
 }
 
+#pragma mark - Key event delegate
+
+- (void)performKeyPress:(int)keyCode withCharacters:(NSString*)characters {
```

Comment:
The key press events are **synthesized** from the callbacks of UIKeyInput in FlutterTextInputPlugin. UIKeyInput provides two callbacks, one when a text is inserted and the other when delete backward is tapped. I am hoping the message structure below is acceptable, in that:
- `keyCode` is set to a value greater than 0 when delete backward is tapped, otherwise it is set to 0
- `characters` is set to the text received by `(void)insertText:(NSString *)text;`, otherwise it is an empty string..